rules/windows/image_load/image_load_side_load_python.yml

```diff
@@ -25,13 +26,18 @@ detection:
         - ImageLoaded|startswith:
               - 'C:\Program Files\Python3'
               - 'C:\Program Files (x86)\Python3'
+              - 'C:\ProgramData\Anaconda3\' # Comment out if you don't use Anaconda in your environment
```

Comment:
Put this is as an optional selection instead

Reply:
I agree, like the cpython below